src/devtools/client/debugger/src/components/shared/Dropdown.css

```diff
@@ -104,8 +104,6 @@
 .dropdown-panel .menu-item {
   padding: 4px 8px;
   color: var(--theme-body-color);
-  line-height: 1.75rem;
-  font-size: 1.1rem;
```

Comment:
it's nice to get rid of arbitrary rem. I worry though when I see devtools changes...